deploy/catalog_resources/community/1.1.0/kiecloud-operator.1.1.0.clusterserviceversion.yaml

```diff
@@ -18,7 +18,10 @@ metadata:
     operator-kiecloud: "true"
 spec:
   displayName: Kie Cloud
-  description: Kie Cloud Operator for deployment and management of RHPAM/RHDM environments.
+  description: |
+    Kie Cloud Operator for deployment and management of RHPAM/RHDM environments.
+
+    **To use the guided installer to provision an environment, open the Installer link, in the links section on the left side of this page.**
```

Comment:
Remove comma after "Installer link", please.

Also this text is used even before subscribing, when there is no installer link. I suggest adding: "After subscribing to the Operator, to use the..."